src/app/components/modals/courseInfoModal/CourseInfoModal.tsx

```diff
@@ -244,45 +245,53 @@ export default function CourseInfoModal({
               {description(data)}
             </Typography>
           </Skeleton>
-          <Skeleton loading={loading} variant="text" width="50%">
-            {!isCSE(course) && !isCustomCourse(course) && (
-              <Typography
-                variant="soft"
-                color="warning"
-                component="p"
-                startDecorator={<WarningAmberRounded color="warning" />}
-              >
-                Warning: quarters offered information is unavailable for this
-                course.
-              </Typography>
-            )}
-            {isCSE(course) && !isOffered(course.quartersOffered, term) && (
-              <Typography
-                variant="soft"
-                color="warning"
-                component="p"
-                startDecorator={<WarningAmberRounded color="warning" />}
-              >
-                Warning: {course.departmentCode} {course.number} is not offered
-                in {` ${term}`}
-              </Typography>
-            )}
-            <Typography component="p">{quartersOffered(data)}</Typography>
-            <Typography component="p">Credits: {credits(data)}</Typography>
-            {data !== undefined && (
-              <>
-                <Typography component="p">{prerequisites(data)}</Typography>
-                <Typography component="p">GE: {ge(data)}</Typography>
-              </>
-            )}
-            {!viewOnly && course.labels && (
-              <SelectedLabels
-                labels={getCourseLabels(course)}
-                handleOpenLabels={handleOpenLabels}
-                ge={course.ge}
-              />
-            )}
-          </Skeleton>
+          {/* Show preqs, ge, past enrollment info, and instructors for official courses*/}
+          {!isCustomCourse(course) ? (
+            <>
+              <Typography component="p">{prerequisites(data)}</Typography>
+              <Typography component="p">GE: {ge(data)}</Typography>
+              <Skeleton loading={enrollLoading} variant="text" width="50%">
+                <div className="flex flex-wrap items-center gap-2">
```

Comment:
<img width="809" alt="image" src="https://github.com/SlugPath/SlugPath/assets/16377965/275bb993-23d2-4892-9787-2297b6788ec2">
Currently it is difficult to visualize which quarters offer the course and which do not.

In the below image, its a bit easier because enrollmentInfo (or your <Chip /> data) is aligned in a grid depending on the quarter the info is in. No need to display quarter title or year, just align the existing Chips so Fall enrollment info is on one row, Winter enrollment info on the next, so on and so fourth.
<img width="1338" alt="Screenshot 2024-02-27 at 16 25 42" src="https://github.com/SlugPath/SlugPath/assets/16377965/c2c7da81-0a4f-40ac-916b-fd4410417df6">